template/.postcssrc.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
   "plugins": {
     // to edit target browsers: use "browserslist" field in package.json
```

Comment:
could you move this line so it's right above `autoprefixer`? That's where it belongs. Thanks! :)

Reply:
I have already modified it.